kmerexpr/multinomial_model.py

```diff
@@ -83,6 +83,7 @@ def __init__(self, x_file=None, y_file=None, beta =1/18, lengths = None, solver_
             y = np.load(y_file)
         self.ymask = y.nonzero() # Need only need self.ynnz and self.xnnz. Throw away the rest?
         self.ynnz = y[self.ymask]
+        self.ynnz = self.ynnz / np.sum(self.ynnz)
```

Comment:
I'm only commenting here, because I'm not 100% sure why this is being normalized this way.

As far as the code goes, I prefer the compound arithmetic/assign when an operation is applied to itself like this one:

```python
self.ynnz /= np.sum(self.ynnz)
```

Same comment for other model.

Reply:
The file has two functions, `logp_grad_old` and `logp_grad`.   Rather than keeping `logp_grad_old` around we should rely on version control and remove it.

Reply:
yes, agreed, and this is my fault for keeping both! Let's delete logp_grad_old

Reply:
also normalizing the count vector doesn't change the [multinomial_model.py] model, but it does affect the way we need to implement some of the optimization methods. So for the sake of simplicity, Fred recommended just normalizing the count vector. I think it makes sense

Reply:
Thanks for the reply.  I don't want to block this PR, but maybe you could explain what's going on when we get back next week.